securesignals-gma/build.gradle

```diff
@@ -23,8 +23,8 @@ dependencies {
 
     // We only compile against the GMA SDK as the exact version being used should be defined by the consuming
     // application.
-    compileOnly 'com.google.android.gms:play-services-ads:21.5.0'
+    compileOnly 'com.google.android.gms:play-services-ads:22.0.0'
```

Comment:
Linting seems a bit aggressive, as forced me to update. Will separately look to remove this lint error